homeassistant/components/esphome/__init__.py

```diff
@@ -54,10 +55,55 @@
 STORAGE_VERSION = 1
 
 
+@attr.s
```

Comment:
Why not use the build-in `@dataclass` instead?

Reply:
Because the rest of this integration uses `attrs` right now as well - there's work in https://github.com/esphome/aioesphomeapi/pull/36 to switch to `dataclasses` which will enable the HA side to switch to it too.

As part of the PR that moves to dataclasses on this side, I will also update this class.

Reply:
Conversion here: https://github.com/home-assistant/core/pull/52305

I will rebase the other PR when one is merged.

Reply:
Merged!

Reply:
Thx! Done